lib/bloc/account/account_bloc.dart

```diff
@@ -221,6 +221,7 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
     int amount,
     bool outgoing, {
     int? channelMinimumFee,
+    bool? channelCreationPossible,
```

Comment:
Can we make it not nullable? Null here doesn't make sense.

Reply:
Added